firstuseauthenticator/templates/reset.html

```diff
@@ -0,0 +1,38 @@
+{% extends "page.html" %}
+
+{% block main %}
+
+<div class="container">
+<form action="{{post_url}}" method="post" role="form">
```

Comment:
We're using Bootstrap 3.3, rather than Bootstrap 4. I think this requires different class annotations for forms to get proper styling. Can you switch it to using horizontal forms, as seen in: https://getbootstrap.com/docs/3.3/css/#forms?